tests/e2e/testfiles/pflex-pscale-values.yaml

```diff
@@ -73,32 +75,49 @@
 - scenario: "Install PowerFlex Driver (With Authorization)"
   paths:
     - "testfiles/storage_csm_powerflex_auth.yaml"
+    - "testfiles/authorization-templates/csm_authorization_proxy_server.yaml"
```

Comment:
Minor nitpick, since the server needs to be installed before the driver, can this be the first testfile in the path? This is how other authorization e2e scenarios are set up and we may want to keep it consistent